### PR TITLE
Fix random consistency

### PR DESCRIPTION
**Pull request by ThomasKluiters**
See #23 

This PR will make it so that the random state is shared across all delegated functions by evopy.py

### CODE REVIEW

File: evopy/evopy.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def __init__(self, fitness_function, individual_length, warm_start=None, generat
         self.maximize = maximize
         self.strategy = strategy
         self.random_seed = random_seed
```
Comment (gandreadis) [inline evopy/evopy.py:42]: Maybe this assignment is no longer necessary then? I think all references to `self.random_seed` have been replaced with a `self.random` reference, so this one can be purely a constructor parameter

**Review by gandreadis — COMMENTED** [review]
Wow, thanks for catching this! 😄 One code-style-related comment.

File: evopy/evopy.py (hunk 1 above)
Comment (ThomasKluiters) [reply evopy/evopy.py:42]: Definitely a good point, I thought about this, but perhaps it's nice for traceability  and debugging: "Which seed is used for this class". As it's actually not trivial to find the seed of a `self.random` without some dirty hacks :(

Comment (gandreadis) [reply evopy/evopy.py:42]: Valid point, that does sound like it might become useful some time.

**Review by gandreadis — APPROVED** [review]

**Pull request merged**